WalletWasabi.Tests/UnitTests/EventSourcing/TestDomain/TestRoundCommandProcessor.cs

```diff
@@ -0,0 +1,118 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using WalletWasabi.EventSourcing.Interfaces;
+using WalletWasabi.EventSourcing.Records;
+
+namespace WalletWasabi.Tests.UnitTests.EventSourcing.TestDomain
+{
+	public class TestRoundCommandProcessor : ICommandProcessor
```

Comment:
I think I already asked re previous version of this, but this is just for illustration purposes, right?

Reply:
yes, mainly as sample domain to allow to unit test the infrastructure